src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/domainevents/DomainEventDescriberTest.kt

```diff
@@ -537,21 +538,24 @@ class DomainEventDescriberTest {
         eventDetails = RequestForPlacementAssessedFactory()
           .withDecision(decision)
           .withDecisionSummary(decision.toString())
+          .withExpectedArrival(LocalDate.of(2024, 5, 3))
+          .withDuration(7)
           .produce(),
       )
     }
 
     val result = domainEventDescriber.getDescription(domainEventSummary)
 
     assertThat(result).isEqualTo(
-      "A request for placement assessment was $decision. The reason was: $decision",
+      "A request for placement assessment was $decision. The reason was: $decision The placement request $expectedTermUsed for Friday 3 May 2024 to Friday 10 May 2024 (1 week)",
```

Comment:
i think there may be a full stop missing here after decision?

Reply:
There is, it's relying (assuming) that the free-text entered is a complete sentence ending with a full stop. 

A few options - 

1. Leave it and hope the text entered is good (possible missing full stop). 
2. Add a full stop (might mean multiple full stops are present in the rendered string).
3. Parse the string and ensure that there is always only 1 full stop present.

Reply:
I'll change the test to include a narrative.

Reply:
Yeah that's tricky. I guess another alternative is to re-order the message so the reason always appears last e.g.

```A request for placement assessment was $decision. The placement request $expectedTermUsed for Friday 3 May 2024 to Friday 10 May 2024 (1 week). The reason was: $decision```

My second preference is to add a fullstop and risk having two.

Reply:
Hopefully covered those both off now.